frontend/src/use-cases/songs/screens/view-songs/components/song-masonry/masonry/useEventListener.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useRef } from "react";
+
+export function useEventListener(eventNames, handler, element = window) {
```

Comment:
I guess you're using this to handle the layout of the grid, but can't you just use CSS rules?

Reply:
From what I have gathered you need javascript to some degree to get a Masonry pattern.

Reply:
This component is of the smaller ones i have found while browsing for a solution to create a Masonry pattern.